docs/CSBR.md

```diff
@@ -503,19 +503,16 @@ If the CA decides that the revocation will have an unreasonable impact on its cu
 
 The CA MUST revoke a Code Signing Certificate within one (1) business day if the Subscriber requests in writing that the CA revoke the Certificate or notifies the CA that the original certificate request was not authorized and does not retroactively grant authorization.
 
-#### 4.9.1.3 Revocation Based on Reported or Detected Compromise or Use in Malware
+#### 4.9.1.3 Revocation Based on Reported or Detected Compromise or Use in Suspect Code
 
-For all incidents involving malware, CAs SHALL revoke the Code Signing Certificate in accordance with and within the following maximum timeframes. Nothing herein prohibits a CA from revoking a Code Signing Certificate prior to these timeframes.
+Except for cases that fall under Section 4.9.1.1, if, while investigating a problem report, the CA determines the Subscriber's Private Key is likely compromised or being used for Suspect Code, the CA SHALL revoke the corresponding Code Signing Certificate in accordance with and within the following maximum time frames. Nothing herein prohibits a CA from revoking a Code Signing Certificate prior to these time frames.
 
-1.  The CA MUST contact the software publisher within one (1) business day after the CA is made aware of the incident.
-2.  The CA MUST determine the volume of relying parties that are impacted (e.g., based on OCSP logs) within 72 hours after being made aware of the incident.
-3.  The CA MUST request the software publisher send an acknowledgement to the CA within 72 hours of receipt of the request.
-  a.  If the publisher responds within 72 hours, the CA and publisher MUST determine a "reasonable date" to revoke the certificate based on discussions with the CA.
-  b.  If CA does not receive a response, the CA must notify the publisher that the CA will revoke in 7 days if no further response is received.
-    i.  If the publisher responds within 7 days, the CA and the publisher will determine a "reasonable date" to revoke the certificate based on discussion with the CA.
-    ii.  If no response is received after 7 days, the CA must revoke the certificate except if the CA has documented proof (e.g., OCSP logs) that the revocation will cause significant impact to the general public.
+1.  The CA MUST contact the Subscriber within 24 hours after the CA received the initial problem report or discovered the incident.
+2.  The CA MUST request the Subscriber send an acknowledgement to the CA within 72 hours of receipt of the request.
+   a.  If the Subscriber responds within 72 hours, the CA and Subscriber MAY determine a "reasonable date" to revoke the certificate. The revocation date MUST NOT be more than 7 calendar days after the incident was discovered.
+   b.  If CA does not receive a response, then the CA MUST revoke the certificate within 24 hours from the end of the response period.

```

Comment:
What is the rationale for allowing less time for revocation if the CA does not receive a response? We may be assuming "if Subscriber does not respond it's malicious intent", but I don't think that's true in all cases.

Reply:
response period is not a defined term, could we update it to "the end of the 72 hours period for the Subscriber to provide an acknowledgement".

Reply:
While I agree it's not true in all cases, I think the rationale for revoking earlier makes sense. If there is indeed signed suspect code, there's a real danger for Relying Parties, and it must be taken care of. subscribers not responding are still looking at pretty much a 4 day revocation window, while for SSL there's a maximum of 5. Imo, the damage suspect code can do, warrants this approach

Reply:
With that explanation, the rationale for revoking earlier does indeed make sense.

While in the previous text "reasonable date" is definitely too open for interpretation, is it appropriate to immediately go to revoking within 7 days in case the Subscriber responds, the timeline that previously applied to case of a non-responding subscriber? 

Can I suggest "SHOULD within 7 days, MUST within 14 days" which would allow CAs to consider specific circumstances, but still putting an upper bound on the timeline compared to "reasonable date"?

Reply:
I'm open to hearing the opinion of others, but I do not feel 14 days is acceptable.